sys/drv/rtl8139_reg.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copied from FreeBSD: sys/dev/rl/if_rlreg.h
+ */
+
+#define RL_COMMAND              0x0037          /* command register */
+
+#define RL_8139_CFG1            0x0052          /* config register #1 */
+
+/*
+ * Command register.
+ */
+#define RL_CMD_RESET            0x0010
+
+/*
+ * Config 1 register
+ */
+#define RL_CFG1_PWRON           0x00
+
+#define RL_TIMEOUT              (10*1000)
```

Comment:
Please add include guard.

Reply:
ok - done. fyi, only i8259.h and isareg.h take care about guard